webpack.config.js

```diff
@@ -0,0 +1,48 @@
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+
+const path = require('path');
+const isDevelopment = process.env.NODE_ENV !== 'production';
```

Comment:
You're not setting the env var in either of your scripts:

```js
const isDevelopment = process.env.NODE_ENV !== 'production';

console.log('process.env.NODE_ENV', process.env.NODE_ENV);
console.log('isDevelopment', isDevelopment);
```

```
$ npm run build

> movie-frog@1.0.0 build D:\git\okhvr\movie-frog
> webpack --config webpack.config.js --mode=production

process.env.NODE_ENV undefined
isDevelopment true
```

```
$ npm run build-dev

> movie-frog@1.0.0 build-dev D:\git\okhvr\movie-frog
> webpack --config webpack.config.js --mode=development

process.env.NODE_ENV undefined
isDevelopment true
```

Reply:
edited, thank you!